### ISSUE TITLE
Can't see status of the devices after homebridge update form 1.2.5 to 1.3.1 

### ISSUE THREAD

**Issue by AlexKusnezov** [oznu/homebridge-gsh#103]
**Describe The Bug:**

After updating the docker container to the latest homebridge version 1.3.1. Homebridge doesn't report the status of the accessories

**To Reproduce:**

Update homebridge to 1.3.1 or higher. Check the accessories in the google home app

**Expected behavior:**

When I turn a light of in homekit, the same light in google home should be updated and report that it is on.

**Logs:**

No waring or reports whatsoever. You just can't see it in the log

```
[3/5/2021, 2:54:28 PM] [Google Smart Home] Received EXECUTE intent
[3/5/2021, 2:54:29 PM] [Google Smart Home] Received QUERY intent
[3/5/2021, 2:54:30 PM] [Google Smart Home] Received EXECUTE intent

```

**Homebridge Config:**

```json
        {
            "name": "Google Smart Home",
            "token": "removeed token",
            "notice": "Keep your token a secret!",
            "twoFactorAuthPin": "removed pin",
            "accessoryFilter": [
                "Hue temperature sensor 1",
                "Balkonlicht",
                "LED Strips",
                "Lautsprecher",
                "Bildschirm Links",
                "Bildschirm Rechts",
                "Rollladen",
                "Markise",
                "Alex Mac Pro",
                "esphome1",
                "Computer Ambilight",
                "Philips TV Ambilight",
                "STSwitch",
                "Security Alarm",
                "removed_OSRAM",
                "13 Deckenleuchte (Wohnzimmer)",
                "15 Stehleuchte",
                "22 Stehleuchte (klein)",
                "CC2531 Router",
                "On-OFF Hue Light 2",
                "On-OFF Hue Light 3",
                "On-OFF Hue Light 4",
                "Rainbow Runner",
                "WC",
                "Waschbecken",
                "29 WC-Leuchte",
                "31 Waschbeckenleuchte",
                "30 Tischfach"
            ],
            "platform": "google-smarthome"
        },
```

**Screenshots:**

After update it looks like this in google home:
![image](https://user-images.githubusercontent.com/33514406/110124060-ee978680-7dc1-11eb-87fe-28efa0d07667.png)
Everything works in the homekit app und alexa and google home can't see the status

**Environment:**

Lastest docker-container with homebridge 1.3.1 on Raspberry Pi 3 b+ 
* **Node.js Version**: 	v14.15.5
* **NPM Version**: 	v6.14.11
* **Homebridge Version**: 
* **Homebridge GSH Plugin Version**: v2.0.1
* **Homebridge Config UI X Plugin Version**:
* **Operating System**:

**Comment by PixelBurstHD** [comment]
Experiencing the same! Let me know if you require any logs to resolve this.

**Comment by rbenomar** [comment]
I'm having the same issue since the last few Homebridge versions.
Temporary fix:
Reverting to version 1.2.5 fixed the problem for me,
You can revert to an older version (if that's an option) using ConfigUI.
Click on the version number on the main page and select v1.2.5 from the dropdown menu.

**Comment by oznu** [comment]
Should be fixed in v1.2.0.
